.github/workflows/scripts/validate-coderabbit.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+echo "Step 1: Fetching all PR reviews with pagination..."
+
+page=1
+all_reviews="[]"
+
+while true; do
+  echo "Fetching page $page..."
+  response=$(curl -s -f -H "Authorization: token $GITHUB_TOKEN" \
+    "https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${PR_NUMBER}/reviews?per_page=100&page=$page") || {
+    echo "Error: Failed to fetch reviews from GitHub API"
+    exit 1
+  }
+  
+  if [ "$(echo "$response" | jq '. | length')" -eq 0 ]; then
+    echo "No more reviews to fetch."
+    break
+  fi
+
+  all_reviews=$(echo "$all_reviews" "$response" | jq -s 'add')
+  page=$((page + 1))
+done
+
+echo "Debug: Combined reviews from all pages:"
+echo "$all_reviews" | jq '.'
+
+reviews="$all_reviews"
+
+echo "Step 2: Parsing the latest review by each user..."
+latest_reviews=$(echo "$reviews" | jq -c '[.[]] | group_by(.user.login) | map(max_by(.submitted_at))') || {
```

Comment:
Should this be a parameter passed to the script?
Looking at the code the script may not be appropriate. It relies on too many GitHub specific values

Reply:
yes because all the data fetched from github api is stored in reviews